fonts.less

```diff
@@ -49,9 +49,10 @@
 .font-face('Main', bold, italic);
 .font-face('Main', normal, italic);
 .font-face('Main', normal, normal);
-// .font-face('Math', bold, italic);
+//.font-face('Math', bold, normal);
+.font-face('Math', bold, italic);
 .font-face('Math', normal, italic);
-// .font-face('Math', normal, normal);
+//.font-face('Math', normal, normal);
```

Comment:
There are no mappings for that font. It's all very confusing. But probably we should add bold italic for `\bm`'s sake, and figure out the duplication later...

Reply:
I'm confused because `KaTeX-Math-Regular` is part of the title for this PR.

I agree though that it's generally confusing.  I wonder if we should stop having separate `-Math-` and `-Main-` fonts.